packages/react-native-template-realm-js/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "react-native-template-realm-js",
+  "version": "0.0.1",
+  "description": "React Native Template Realm JS",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/realm/realm-js.git",
+    "directory": "packages/react-native-template-realm-js"
+  },
+  "author": {
+    "name": "Realm",
+    "email": "help@realm.io",
+    "url": "https://realm.io"
+  },
+  "license": "MIT"
```

Comment:
```suggestion
  "license": "Apache-2"
```

Reply:
`Apache-2.0` complies to the [SPDX](https://spdx.org/licenses/) format

Reply:
Moreover, please add https://www.apache.org/licenses/LICENSE-2.0.txt as `LICENSE`